lib/rpcc_dpdk/rte_flow_ndn.c

```diff
@@ -842,3 +842,164 @@ rpc_rte_flow_release_united_items(rcf_rpc_server       *rpcs,
 
     RETVAL_VOID(rte_flow_release_united_items);
 }
+
+static const char *
+tarpc_rte_flow_indir_action_conf2str(
+    const struct tarpc_rte_flow_indir_action_conf *conf,
+    te_string *conf_str)
+{
+    if (conf_str == NULL)
+        return "{ NULL }";
+
+    te_string_reset(conf_str);
+    te_string_append(conf_str, "{ ingress: %s, egress: %s, transfer: %s }",
+                     (conf->ingress) ? "TRUE" : "FALSE",
+                     (conf->egress) ? "TRUE" : "FALSE",
+                     (conf->transfer) ? "TRUE" : "FALSE");
+
+    return conf_str->ptr;
+}
+
+rpc_rte_flow_action_handle_p
+rpc_rte_flow_action_handle_create(rcf_rpc_server *rpcs, uint16_t port_id,
+                                  tarpc_rte_flow_indir_action_conf *conf,
+                                  rpc_rte_flow_action_p action,
+                                  struct tarpc_rte_flow_error *error)
+{
+    tarpc_rte_flow_action_handle_create_in   in = {};
+    tarpc_rte_flow_action_handle_create_out  out = {};
+    te_string                                conf_str = TE_STRING_INIT;
+    te_log_buf                              *tlbp;
+
+    if (conf == NULL)
```

Comment:
NULL must be supported. RPC wrapper must allow to call the function with NULL value.